src/main/java/de/tum/in/www1/artemis/security/jwt/JWTCookieService.java

```diff
@@ -56,12 +58,13 @@ public ResponseCookie buildLogoutCookie() {
      */
     private ResponseCookie buildJWTCookie(String jwt, Duration duration) {
 
+        // TODO - Remove cypress workaround once cypress uses https and find a better solution for testing locally in Safari
         Collection<String> activeProfiles = Arrays.asList(environment.getActiveProfiles());
-        boolean isCypress = activeProfiles.contains(CYPRESS_PROFILE);
+        boolean isSecure = !activeProfiles.contains(CYPRESS_PROFILE) && !activeProfiles.contains(DEVELOPMENT_PROFILE);
 
         return ResponseCookie.from(JWT_COOKIE_NAME, jwt).httpOnly(true) // Must be httpOnly
                 .sameSite("Lax") // Must be Lax to allow navigation links to Artemis to work
```

Comment:
```suggestion
        return ResponseCookie.from(JWT_COOKIE_NAME, jwt)
                .httpOnly(true) // Must be httpOnly
                .sameSite("Lax") // Must be Lax to allow navigation links to Artemis to work
```

Reply:
Spotless does not like your change:
![grafik](https://github.com/ls1intum/Artemis/assets/26540346/3b21636c-13b4-4598-a77a-8d56dfcc4302)

Reply:
I will revert the commit